formats/email_alert_signup/publisher/details.json

```diff
@@ -13,17 +13,7 @@
       "minProperties": 1,
       "properties": {
         "tags": {
-          "type": "object",
-          "description": "DEPRECATED: The tags used to match subscribers lists",
```

Comment:
I think removed from hmrc manuals in a separate PR makes a lot of sense, yes.  Will give us a chance to make sure there aren't any copies lurking in the content-store that still have it, which we can republish if needs be.